docs/getting-started/creating-users.md

```diff
@@ -0,0 +1,39 @@
+## Auth Concepts
+
+Fleet auth has a concept of Users, AuthCerts, Roles and Policies.
```

Comment:
Just "Fleet has the concept of Users, Roles, Policies and Authentication Certificates"

Maybe a diagram here to show how they interact would be good.